src/parallel/parallel.hpp

```diff
@@ -194,10 +199,10 @@ class ParallelRobinHoodHashTable {
     find(KeyType key);
 
 private:
-    std::vector<ParallelBucket> buckets_;
+    std::vector<ParallelBucket> buckets_ = std::vector<ParallelBucket>(1);
     std::vector<SegmentLock> segment_locks_;
-    size_t length_;
-    size_t capacity_;
+    size_t length_ = 0;
+    size_t capacity_ = 1;
```

Comment:
Consider starting at a higher capacity. I know the sequential one starts at 1. But even that isn't a great idea. I guess we can just prove that it works for any size before you bother changing it.